evals/prompts/agent_find_and_update_prompts/simple_prompt.py

```diff
@@ -0,0 +1,60 @@
+from typing import List
+from evals.eval_types import AgentFindAndUpdatePromptGenerator
+from evals.test_cases.agent_find_and_update_tests.simple import Cell
+
+__all__ = ['simple_prompt_v1']
+
+class _SimplePromptV1(AgentFindAndUpdatePromptGenerator):
+    prompt_name = "simple_prompt_v1"
+
+    def get_prompt(self,  user_input: str, initial_notebook_state: List[Cell]) -> str:
+        return f"""You're an expert python data scientist working in Jupyter Lab. Your job is to help your colleagues update their code in Jupyter. 
+    
+Your colleague is going to provide you: 
+1. Their Jupyter Notebook in the format
+[
+    {{
+        cell_type: Literal['code', 'markdown']
+        id: str
+        code: str
+    }},
+    {{
+        cell_type: Literal['code', 'markdown']
+        id: str
+        code: str
+    }},
```

Comment:
Probably nothing, but do we need to have this example object twice? I would guess the fact that it's in an array should make it obvious that there'd be multiple items. 
```suggestion
    {{
        cell_type: Literal['code', 'markdown']
        id: str
        code: str
    }},
```